modules/ROOT/partials/configuration/tinycomments_fetch_author_info.adoc

```diff
@@ -0,0 +1,61 @@
+[[tinycomments_fetch_author_info]]
+== `+tinycomments_fetch_author_info+`
+
+The {pluginname} plugin uses the `+tinycomments_fetch_author_info+` callback function to provide author details for comments displayed in the editor.
+
+This function is called whenever a comment is rendered, and it allows the editor to display the author’s name and avatar based on the current user.
+
+The `+tinycomments_fetch_author_info+` function is passed a request (`+req+`) object as the first parameter, and a `+done+` callback as the second.
+
+The function must call `+done+` with an object containing the following properties:
+
+* `+author+` (`+string+`): A unique identifier for the user.
```

Comment:
This one is necessary, the other two properties are optional. I think this is worth mentioning.

Reply:
Thanks @hamza0867 Updated in [commit](https://github.com/tinymce/tinymce-docs/pull/3675/commits/df039f99a9cac36bdccc09de5ade302cf1adaa61)